assets/javascripts/discourse/controllers/docs-index.js

```diff
@@ -292,7 +292,9 @@ export default Controller.extend({
   @action
   updateSelectedCategories(category) {
     const filterCategories =
-      category.id === this.filterCategories ? null : category.id;
+      parseInt(category.id, 10) === parseInt(this.filterCategories, 10)
```

Comment:
I don't think `parseInt(category.id, 10)` is necessary because `category.id` should already be an integer.

Reply:
Yeah, good point, updated that line.